Formula/xplanet.rb

```diff
@@ -44,6 +45,9 @@ class Xplanet < Formula
     sha256 "3f95ba8d5886703afffdd61ac2a0cd147f8d659650e291979f26130d81b18433"
   end
 
+  # Support for giflib 5 is incomplete
```

Comment:
Was this reported?

Also, a patch of that size needs to go in formula-patches.